core/src/modules/modelName.ts

```diff
@@ -1,7 +1,7 @@
 import { Model } from "sequelize-typescript";
-import { LoggedModel } from "../classes/loggedModel";
+import { CommonModel } from "../classes/commonModel";
 
-export function modelName(instance: Model | typeof LoggedModel) {
+export function modelName(instance: Model | typeof CommonModel) {
```

Comment:
Not sure this adds any value but:
```suggestion
export function modelName<T>(instance: Model | CommonModel<T>) {
```

Reply:
Since T will be optional, might as well!